healthcheck.go

```diff
@@ -17,8 +17,8 @@ import (
 // HTTP path to check the health of a cluster using Elasticsearch API
 const pathHealth = "/_cluster/health"
 
-// MsgHealthy Check message returned when elasticsearch is healthy
-const MsgHealthy = "elasticsearch is healthy"
+// MsgHealthy Check message returned when elasticsearch is healthy and indexed
+const MsgHealthy = "elasticsearch is indexed and healthy"
```

Comment:
I wonder if perhaps saying

```suggestion
// MsgHealthy Check message returned when elasticsearch is healthy and the required indexes exist
const MsgHealthy = "elasticsearch is healthy and the required indexes exist"
```

would be more accurate. When reading `is indexed` I think of the data being correctly indexed rather than the indexes themselves existing.